src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodRepository.scala

```diff
@@ -50,10 +50,15 @@ object MethodRepository {
     def toShortString: String = s"Method($namespace,$name,$snapshotId)"
   }
 
+  case class AgoraConfigurationShort(
+    name: String,
+    rootEntityType: String,
+    methodRepoMethod: AgoraMethod,
+    namespace: String)
+
   object Method {
```

Comment:
This thing should probably be called `AgoraMethod` now, or be replaced wholesale by `AgoraEntity` from Rawls... along with its sibling `Configuration`

I'm going for maintenance-adequate with this PR though, not an opportunistic refactoring